src/MahApps.Metro/MahApps.Metro.Shared/Controls/MetroWindow.cs

```diff
@@ -325,6 +330,21 @@ public bool IgnoreTaskbarOnMaximize
         }
 
         /// <summary>
+        /// Gets or sets resize border thickness
+        /// </summary>
+        public Thickness ResizeBorderThickness
+        {
+            get { return (Thickness)this.GetValue(ResizeBorderThicknessProperty); }
+            set { this.SetValue(ResizeBorderThicknessProperty, value); }
+        }
+
+        /// <summary>
+        /// Using a DependencyProperty as the backing store for ResizeBorderTickness.  This enables animation, styling, binding, etc...
+        /// </summary>
+        public static readonly DependencyProperty ResizeBorderThicknessProperty =
+            DependencyProperty.Register(nameof(ResizeBorderThickness), typeof(Thickness), typeof(MetroWindow), new PropertyMetadata(BorderlessWindowBehavior.GetDefaultResizeBorderThickness()));
+
```

Comment:
this should be `WindowChromeBehavior` instead of the old `BorderlessWindowBehavior`